crates/red_knot_python_semantic/src/types/class.rs

```diff
@@ -341,6 +341,13 @@ impl<'db> ClassType<'db> {
             }
         }
 
+        if self
+            .iter_mro(db)
+            .any(|base| matches!(base, ClassBase::Dynamic(DynamicType::Any)))
```

Comment:
Originally I wanted to allow all dynamic bases, but that leads to a lot of problems. It makes all kinds of types (like `str`) assignable to everything, if they have `@Todo` bases in their MRO.

Reply:
> Ideally I feel like we'd at least treat `Any` and `Unknown` the same way

I know, but I'm afraid then we might treat `C` as being assignable to everything in cases like below, which have nothing to do with not-yet-implemented features:

```py
from non_installed_dependency import SomeBase

class C(SomeBase): ...

x: int = C()
```

Reply:
That feels like a feature rather than a bug, though? You'd get one diagnostic from the fact that we can't resolve the import. There's no need to have cascading diagnostics after that because of the single error. Maybe when `non_installed_dependency` _is_ installed, it turns out that `SomeBase` is just a type alias for `int`!

Reply:
I agree with Alex. Treating unknown imports as the dynamic type (same as Any) is correct and the intended behavior, including in that example.

Reply:
That makes sense, thanks! This helped remove even more false positives.